lib/pcap/source.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Membrane.Element.Pcap.Source do
+  use Membrane.Element.Base.Source
+
+  alias Membrane.Buffer
+  alias Membrane.Event.EndOfStream
+
+  def_output_pads output: [
+                    caps: :any
+                  ]
+
+  def_options packet_transformer: [
+                type: :function,
+                spec: (any() -> Buffer.t()),
```

Comment:
Shouldn't it go from `ExPcap.Packet`?

Reply:
That's a good addition. I will implement it.